### PR TITLE
Remove title def from owned categories

### PR DESCRIPTION
**Pull request by otter-computer**

### CODE REVIEW

File: .github/DISCUSSION_TEMPLATE/profile.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-title: "<title>"
```
Comment (Kraviscloud) [inline .github/DISCUSSION_TEMPLATE/profile.yml:1]: gp

Comment (Kraviscloud) [inline .github/DISCUSSION_TEMPLATE/profile.yml:1]: Duplicate of #

File: .github/DISCUSSION_TEMPLATE/profile.yml [hunk 2]
```diff
@@ -1,4 +1,3 @@
...
 labels: [Profile]
```
Comment (Kraviscloud) [inline .github/DISCUSSION_TEMPLATE/profile.yml:1]: #

File: .github/DISCUSSION_TEMPLATE/profile.yml [hunk 3]
```diff
@@ -1,4 +1,3 @@
...
 body:
 - type: dropdown
```
Comment (Kraviscloud) [inline .github/DISCUSSION_TEMPLATE/profile.yml:3]: 1

File: .github/DISCUSSION_TEMPLATE/profile.yml [hunk 4]
```diff
@@ -1,4 +1,3 @@
-title: "<title>"
 labels: [Profile]
 body:
```
Comment (Kraviscloud) [inline .github/DISCUSSION_TEMPLATE/profile.yml:2]: 1

File: .github/DISCUSSION_TEMPLATE/profile.yml (hunk 3 above)
Comment (Kraviscloud) [inline .github/DISCUSSION_TEMPLATE/profile.yml:3]: 1

**Comment by devsuperstar121** [comment]
sure

**Comment by emmyforeignet** [comment]
a76157efbd1edcd62570fb4e3f5c98415be6e46a

**Review by emmyforeignet — COMMENTED** [review]
a76157efbd1edcd62570fb4e3f5c98415be6e46a
#48808

**Review by emmyforeignet — COMMENTED** [review]
gh pr checkout 48808

**Comment by fd98ac6** [comment]
ghostinhershell-category-form-update

**Pull request merged**